xml/System.Web.Services.Description/HttpOperationBinding.xml

```diff
@@ -27,7 +27,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This class respresents an extensibility element in Web Services Description Language (WSDL) specific to HTTP. It specifies the relative URL of an operation using as the base the location specified by an <xref:System.Web.Services.Description.HttpAddressBinding>. For more information about specifying protocols for XML Web services, see [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c). For more information about WSDL, see the specification at http://www.w3.org/TR/wsdl/.  
+ This class respresents an extensibility element in Web Services Description Language (WSDL) specific to HTTP. It specifies the relative URL of an operation using as the base the location specified by an <xref:System.Web.Services.Description.HttpAddressBinding>. For more information about specifying protocols for XML Web services, see [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c). For more information about WSDL, see the [WSDL](https://www.w3.org/TR/wsdl/) specification.  
```

Comment:
[&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c) --> \XML Web Services Using ASP.NET\(https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ba0z6a33(v=vs.100))

Reply:
I've opened a separate issue to take care of this site wide: #1608